exercises/grade-school/src/test/kotlin/SchoolTest.kt

```diff
@@ -57,8 +57,8 @@ class SchoolTest {
         school.add("Bradley", 5)
         school.add("Jeff", 1)
 
-        val expected = mapOf(5 to listOf("Franklin", "Bradley"), 1 to listOf("Jeff"))
-        assertEquals(expected, school.db())
+        val expected = mapOf(5 to listOf("Franklin", "Bradley"))
```

Comment:
Are you referring to [this](https://github.com/exercism/problem-specifications/blob/master/exercises/grade-school/canonical-data.json#L50) test? I see that this case requires results to be sorted in alphabetical order.